src/runtime/utils.ts

```diff
@@ -0,0 +1,60 @@
+import * as fs from "fs/promises";
+import { PathLike } from "fs";
+import { promisify } from "util";
+import {ChildProcess, spawn as _spawn} from "child_process";
+import { spawn as _asyncSpawn, Output }  from "promisify-child-process";
+import rimraf from "rimraf";
+// @ts-ignore
+import getBinary from "near-sandbox/getBinary";
+import fs_extra from "fs-extra";
+
+export const rm = promisify(rimraf);
+export const sandboxBinary: () => string = () => getBinary().binaryPath;
+
+
+export async function exists(d: PathLike): Promise<boolean> {
```

Comment:
```suggestion
async function exists(d: PathLike): Promise<boolean> {
```

Reply:
This is used by the sever so it needs to be exported.